libs/GraphicECS/SFML/Resources/SoundResource.hpp

```diff
@@ -10,37 +10,38 @@
 
 #include <SFML/Audio.hpp>
 #include <filesystem>
+#include "Error/Error.hpp"
 #include "Resource/Resource.hpp"
 #include <unordered_map>
-#include "Error/Error.hpp"
 
 namespace graphicECS::SFML::Resources
 {
     /// @brief This resource class stores a map of Sounds SFML.
     class SoundResource : public ecs::Resource {
       public:
         /// @brief Enumeration of all available Sounds
-        enum soundName_e { UNDEFINED, SHOOT, DEATH, EXPLOSION };
+        enum sound_e { UNDEFINED, SHOOT, DEATH, EXPLOSION };
 
         /// @brief Name of map which contains Sounds.
-        using SoundsList = std::unordered_map<soundName_e, std::shared_ptr<sf::Sound>>;
+        using SoundsList = std::unordered_map<sound_e, std::shared_ptr<sf::SoundBuffer>>;
 
         /// @brief Add a Sound from it's Sound Path passed as parameter
```

Comment:
It's the constructor who calls a function who's adding, the constructor do not directly add in the map (Doc)

Reply:
improve in #281